src/shared/components/idme/idme.js

```diff
@@ -4,14 +4,28 @@ import troopImage from 'images/Troop.png';
 import styles from './idme.css';
 
 class Idme extends Component {
-  idMe = () => {
-    window.open(`${config.idmeOAuthUrl}?client_id=${config.idmeClientId}&redirect_uri=${config.host}/profile/verify&response_type=token&scope=military&display=popup', '', 'scrollbars=yes,menubar=no,status=no,location=no,toolbar=no,width=750,height=780,top=200,left=200`);
+  onKeyUp = (event) => {
+    if (event.key === 'Enter') {
+      this.idMe();
+    }
```

Comment:
@kylemh Looks like there was some refactoring done here,  where's the `idMe` method located at in this class now?

Reply:
a11y required onKey_ method for non-button/link onClick handlers.

I'm not too familiar with the workings of onKey presses. I'm assuming a good way to move forward would be to give it a tab index to ensure that people are :focus on the IDME button before they click enter

Reply:
In the interest of getting the PR merged, I'll reactivate the rule as a warning. We should eventually create an Epic to tackle accessibility for the website.